src/screens/OtherProfile/index.js

```diff
@@ -408,12 +409,16 @@ const OtherProfile = () => {
     return (
       <>
         <View style={styles.headerImageContainer}>
-          <Image
-            style={styles.profileImage}
-            source={{
-              uri: dataMain.profile_pic_path ?? DEFAULT_PROFILE_PIC_PATH
-            }}
-          />
+          <View style={{marginRight: 22}}>
+            <CircleGradient fill={dataMain.karma_score} size={100} width={3}>
+              <Image
+                style={styles.profileImage}
+                source={{
+                  uri: dataMain.profile_pic_path ?? DEFAULT_PROFILE_PIC_PATH
+                }}
+              />
+            </CircleGradient>
+          </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `CircleGradient` component is used here to wrap the profile image and display a circular gradient based on the user's karma score. However, there seems to be no error handling or fallback in case `dataMain.karma_score` is null or undefined. This could potentially lead to runtime errors. Consider adding a default value or some form of error handling for `karma_score`.
```diff
- <CircleGradient fill={dataMain.karma_score} size={100} width={3}>
+ <CircleGradient fill={dataMain.karma_score ?? 0} size={100} width={3}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
done